scripts/bgpv4_forwarding.go

```diff
@@ -0,0 +1,216 @@
+package main
+
+import (
+	"log"
+	"time"
+
+	"github.com/open-traffic-generator/snappi/gosnappi"
+)
+
+// hostname and interfaces of ixia-c-one node from containerlab topology
+const (
+	otgHost  = "https://clab-ixia-c-ixia-c-one"
+	otgPort1 = "eth1"
+	otgPort2 = "eth2"
+)
+
+var (
+	pktCount = 1000
+	routeCount = int32(5)
+	srcMAC = "00:00:01:01:01:01"
+	srcIP = "1.1.1.2"
+	srcGateway = "1.1.1.1"
+	srcPrefix = 24
+	srcAS = 2222
+	dstMAC = "00:00:02:02:02:02"
+	dstIP = "2.2.2.2"
+	dstGateway = "2.2.2.1"
+	dstPrefix = 24
+	dstAS = 3333
+
+)
+
+func main() {
+
+	api, config := newConfig()
+
+	// push traffic configuration to otgHost
+	res, err := api.SetConfig(config)
+	checkResponse(res, err)
+
+	// start protocol configured devices
+	ps := api.NewProtocolState().
+		SetState(gosnappi.ProtocolStateState.START)
+	res, err = api.SetProtocolState(ps)
+	checkResponse(res, err)
+
+	// fetch Bgpv4 metrics and wait for sessions to be up and routes count to be correct
+	mr := api.NewMetricsRequest()
+	mr.Bgpv4()
+	waitFor(
+		func() bool {
+			res, err := api.GetMetrics(mr)
+			checkResponse(res, err)
+
+			bgp1m := res.Bgpv4Metrics().Items()[0]
```

Comment:
Maybe a better name here ? Could not follow what bgp1m and bgp2m refer to.

Reply:
renaming done